lib/mediawiki/query/lists/search.rb

```diff
@@ -36,6 +36,24 @@ def get_search_results(search_value, namespace = 0)
           query_ary(params, 'search', 'title')
         end
 
+        # Gets an array containing page titles that matched the search.
+        # @param search_value [String] The thing to search for.
+        # @param namespace [Fixnum] The namespace to search in. Defaults to 0 (the main namespace).
```

Comment:
`Fixnum` is actually not a thing anymore. This should be changed to `Integer`. It's outdated throughout the codebase, true, but new code should use the correct class name now.